torch/tensor.py

```diff
@@ -394,8 +395,10 @@ def lu(self, pivot=True, get_infos=False):
         else:
             return LU, pivots
 
-    def stft(self, n_fft, hop_length=None, win_length=None, window=None,
-             center=True, pad_mode='reflect', normalized=False, onesided=True):
+    def stft(self, n_fft: int, hop_length: Optional[int] = None,
```

Comment:
What are stft and istft explicitly registered on tensor like this instead of just relying on native_functions.yaml to generate their tensor methods?

Reply:
For `stft`, it's because it uses `pad` which isn't a fully native function:
https://github.com/pytorch/pytorch/blob/356aa54694cc4f1c9599fdf8320a4759aa63a12f/torch/nn/functional.py#L3487

For `istft` I can't see a reason. It has the same signature as the native function and doesn't do any work in python.

Reply:
Got it. Not a regression, anyway, so no need to dig in further.